samples/dlp_inspect_file.js

```
@@ -0,0 +1,91 @@
+const DLP = require('@google-cloud/dlp');
+
+// Import other required libraries
+const fs = require('fs');
+const mime = require('mime');
+
+/**
+ * Inspects the specified file.
+ *
+ * @param projectId {string} Your Google Cloud Project ID.
+ * @param filepath {string} The path to the file to inspect.
+ * @param mimeType {string} The mimeType file.
+ * @returns {object} The InspectContentResponse.
+ */
+async function main(
+  projectId = 'YOUR_PROJECT_ID',
+  filepath = 'path/to/image.png',
+  mimeType = 'IMAGE_PNG'
+) {
+  // Instantiates a client
+  const dlp = new DLP.DlpServiceClient();
+
+  // Get the bytes of the file
+  const fileBytes = Buffer.from(fs.readFileSync(filepath)).toString('base64');
+
+  // Construct request
+  const request = {
+    parent: dlp.projectPath(projectId),
+    item: {
+      byteItem: {
+        type: mimeType,
+        data: fileBytes,
+      },
+    },
+    inspectConfig: {
+      // The infoTypes of information to match
+      infoTypes: [
+        {name: 'PHONE_NUMBER'},
+        {name: 'EMAIL_ADDRESS'},
+        {name: 'CREDIT_CARD_NUMBER'},
+      ],
+      // The minimum likelihood required before returning a match
+      minLikelihood: 'LIKELIHOOD_UNSPECIFIED',
+      // Whether to include the matching string
+      includeQuote: true,
+      limits: {
+        // The maximum number of findings to report per request
+        // (0 = server maximum)
+        maxFindingsPerRequest: 0,
+      },
+    },
+  };
+
+  // Run request
+  try {
+    const [inspectResponse] = await dlp.inspectContent(request);
+    const findings = inspectResponse.result.findings;
+    console.log(`Findings: ${!findings.length ? 'None' : ''}`);
+    findings.forEach(finding => {
+      console.log(`\tQuote: ${finding.quote}`);
+      console.log(`\tInfo type: ${finding.infoType.name}`);
+      console.log(`\tLikelihood: ${finding.likelihood}`);
+    });
+    return inspectResponse;
+  } catch (err) {
```

I don't think the try/catch here is neccessary, and adds a ton of bloat to our samples.  Why not just let it throw?

---

I dig that

---

Done